benchmarks/api/fabric/workloads/compositeTx/read-write-assets.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict';
+const bytes = (s) => {
```

Comment:
This is already defined in helper.js could you migrate all use to that version (batch-create-asset.js, create-asset.js, create-private-asset.js as well)